### DISCUSSION TITLE
Search icon not working

### DISCUSSION THREAD

**Discussion by CodeWithYashu** [bgrand-ch/grapesjs-icons#4] [Q&A]
Search icons not working

**Comment by bgrand-ch** [comment]
Hello, if you try to do a search, a warning message appears in the console:
https://github.com/bgrand-ch/grapesjs-icons/blob/862d854d28390442d6e679f937390db62d1697e9/src/utils/listener.ts#L117

I'll be adding this functionality shortly. Your help is most welcome.

**Comment by bgrand-ch** [comment]
@CodeWithYashu ℹ️ WIP #8
